src/components/Button.js

```diff
@@ -21,11 +25,40 @@ const Button = React.createClass({
 
   getDefaultProps () {
     return {
+      isActive: false,
       primaryColor: StyleConstants.Colors.PRIMARY,
       type: 'primary'
     };
   },
 
+  _renderButtonContent () {
+    const circleStyles = {
+      display: 'inline-block',
+      verticalAlign: 'middle',
+      borderRadius: '100%',
```

Comment:
I'm not sure what you mean. The height & width are set to the same amount. On the `Loader` if you make the transparent boarders a color, you can see the lopsided spin. It's tough to capture via gif but I can show you on my computer.
![i2qxhlf3dv](https://cloud.githubusercontent.com/assets/1916697/13718216/3735ccfc-e7a5-11e5-9992-8ed96199ea5f.gif)